pkg/cleanup/types/built_in.go

```diff
@@ -113,6 +113,9 @@ func scServiceInstanceCleanup(spec, actual *unstructured.Unstructured) (*unstruc
 	instanceSpec.ObjectMeta.Finalizers = instanceActual.ObjectMeta.Finalizers
 
 	instanceSpec.Spec.ExternalID = instanceActual.Spec.ExternalID
+	if instanceActual.Spec.UserInfo != nil {
+		instanceSpec.Spec.UserInfo = instanceActual.Spec.UserInfo.DeepCopy()
+	}
```

Comment:
Anund mentioned that because we only care about the parameters section, we could probably copy a larger chunk of things/code could be simpler.

Reply:
yeah I can imagine making a deep copy of the entire spec and just overwriting parameters + parametersFrom